components/Form/Form.js

```diff
@@ -7,9 +7,7 @@ export default function Form({ onSubmit, customer }) {
     <>
       <Header />
       <FormContainer>
-        <FormStyle
-          onSubmit={(event) => onSubmit(event, customer && customer.id)}
-        >
+        <FormStyle onSubmit={(event) => onSubmit(event)}>
```

Comment:
The function you pass as a prop should be named 'handleSubmit' and then your form should look like:
``<FormStyle onSubmit={handleSubmit}>...``